components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIAdminImpl.java

```diff
@@ -311,6 +312,18 @@ public void deleteCategory(String categoryID, String username) throws APIManagem
         return apiMgtDAO.getAllCategories(tenantId);
     }
 
+    public List<APICategory> getAPICategoriesOfTenant(int tenantId) throws APIManagementException {
```

Comment:
Shall we add the override annotation to this method to indicate this method overrides the method declared in the APIAdmin.java interface?

Reply:
Fixed with d1a24a8